src/js/containers/search/visualizations/geo/GeoVisualizationSectionContainer.jsx

```diff
@@ -254,28 +260,52 @@ const GeoVisualizationSectionContainer = (props) => {
         setMapLayer(layer);
         setRenderHash(`geo-${uniqueId()}`);
         setLoadingTiles(true);
+        useEffectRef.current.loadingTiles = true;
+
 
         logMapLayerEvent(layer);
     };
 
+    const calculateCenterPoint = (location) => {
+        if (location) {
+            let locationRequest = performCountryGeocode(location);
+            locationRequest.promise
+                .then((res) => {
+                    setCenter(res.data?.features[0]?.center ? res.data?.features[0]?.center : USACenterPoint);
+                })
+                .catch((err) => {
+                    if (!isCancel(err)) {
+                        console.log(err);
+                        locationRequest = null;
+                    }
+                });
+        }
+        else {
+            setCenter(USACenterPoint);
+        }
+    };
+
     const mapScopeLogic = (type) => {
         const selectedLocationByType = type === "pop" ? "selectedLocations" : "selectedRecipientLocations";
         // there is only 1 item, place of performance
         if (props.reduxFilters[selectedLocationByType].size === 1) {
             const onlyObject = props.reduxFilters[selectedLocationByType].first().filter;
+            setSingleLocationSelected(onlyObject);
             if (onlyObject.district_current || onlyObject.district_original) {
```

Comment:
should these all be onlyObject with a null check

ex onlyObject?.district_current

Reply:
I don't think so.  The `if (onlyObject.district_current || onlyObject.district_original)` condition was added for a previous ticket and not touched for this ticket.  So this condition is already in prod.